src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Figure[] value = Figure.values();
+        int figure = random.nextInt(FIGURE_COUNT);
+        return value[figure];
+    }
+
+    public Figure getDefaultFigure() {
+        return Figure.CIRCLE;

```

Comment:
Return here an object of the circle class with the parameters that are specified in the conditions of the problem